multi-educator/src/Dscp/MultiEducator/Launcher/Mode.hs

```diff
@@ -128,15 +128,15 @@ loadEducator login mpassphrase = do
         sel -> error $ "unknown AppDir type: " <> fromString sel
     ctx <- ask
     let resources = ctx ^. lensOf @MultiEducatorResources
-        (MultiEducatorKeyParams keyFile) = multiEducatorConfig ^. sub #educator . option #keys
+        (MultiEducatorKeyParams keyDir) = multiEducatorConfig ^. sub #educator . option #keys
         db = resources ^. lensOf @SQL
     -- set the DB schema name and create it if it's not ready
     setSchemaName db ("educator_" <> login)
     prepareEducatorSchema db
-    liftIO $ createDirectoryIfMissing True (appDir </> keyFile)
+    liftIO $ createDirectoryIfMissing True (appDir </> keyDir)
     -- read key from file and creates one if it does not exist yet
     let keyParams = finaliseDeferredUnsafe $ mempty
-            & option #path       ?~ Just keyFile
+            & option #path       ?~ Just (keyDir </> toString login <.> "key")
```

Comment:
As it is now there is the problem that a few lines above `createDirectoryIfMissing` is called on `appDir </> keyDir`, which is not the one used here in `keyParams`, and so that with the current `scripts/launch/node.sh` configuration the directory created would be `./run/run/educator.key` and the keyfile path that `linkStore` will try to use ends up being `run/educator.key/{educatorId}.key`.

If we want these keys to be in a `tmp` directory, I would suggest to:
- change in `scripts/launch/node.sh` the option `--educator-key-dir` to be just `$tmp_files`
- change the line above this one to use `createDirectoryIfMissing` on `keyDir`
- leave this line like this
This way the directory to create if necessary (`$tmp_files` => `run/tmp/multi-educator`) will contain the keyfiles with the path `run/tmp/multi-educator/{educatorId}.key`. 

If instead you want these to be persistent I would suggest to:
- change in `scripts/launch/node.sh` the option `--educator-key-dir` to be `$files/multieducator`
- change the line above this one to use `createDirectoryIfMissing` on `keyDir`
- leave this line like this
- add `run/multieducator/` in `.gitignore`
This way the keyfiles will be created as `./run/multieducator/{educatorId}.key` and the directory containing them (`./run/multieducator`) will be created if necessary.